src/daemon/scheduler.rs

```diff
@@ -169,230 +126,161 @@ fn make_cas_event_triggers(event_queue: Arc<MessageQueue>, ca_manager: Arc<CaMan
-        let requeue_time = if test_mode_enabled() {
-            requeue_time_test()
-        } else {
-            requeue_time()
-        };
-
-        error!("Failed to publish for '{}' will reschedule, error: {}", ca, e);
-        event_queue.reschedule_sync_repo(ca, requeue_time);
+            sleep(Duration::from_millis(500)).await;
+        }
     }
-}
 
-/// Try to synchronize a CA with its parents, reschedule if this fails
-async fn try_sync_parent(
-    event_queue: &Arc<MessageQueue>,
-    ca_manager: &CaManager,
-    ca: Handle,
-    parent: ParentHandle,
-    actor: &Actor,
-) {
-    info!("Synchronize CA '{}' with its parent '{}'", ca, parent);
-    if let Err(e) = ca_manager.ca_sync_parent(&ca, &parent, actor).await {
-        let requeue_time = if test_mode_enabled() {
-            requeue_time_test()
-        } else {
-            requeue_time()
-        };
-
-        error!(
-            "Failed to synchronize CA '{}' with its parent '{}', error: {}",
-            ca, parent, e
-        );
-        event_queue.reschedule_sync_parent(ca, parent, requeue_time);
-    }
-}
-
-fn make_cas_republish(ca_server: Arc<CaManager>, event_queue: Arc<MessageQueue>) -> ScheduleHandle {
-    SkippingScheduler::run(
-        SCHEDULER_INTERVAL_SECONDS_REPUBLISH,
-        "CA certificate republish",
-        move || {
-            let rt = Runtime::new().unwrap();
-            rt.block_on(async {
-                debug!("Triggering background republication for all CAs, note this may be a no-op");
-                match ca_server.republish_all().await {
-                    Err(e) => error!("Background republishing of MFT and CRLs failed: {}", e),
-                    Ok(cas) => {
-                        for ca in cas {
-                            info!("Re-issued MFT and CRL for CA: {}", ca);
-                            event_queue.schedule_sync_repo(ca);
-                        }
-                    }
-                }
-            })
-        },
-    )
-}
-
-fn make_cas_objects_renew(ca_server: Arc<CaManager>, actor: Actor) -> ScheduleHandle {
-    SkippingScheduler::run(SCHEDULER_INTERVAL_SECONDS_ROA_RENEW, "CA ROA renewal", move || {
-        let rt = Runtime::new().unwrap();
-        rt.block_on(async {
-            debug!(
-                "Triggering background renewal for about to expire objects issued by all CAs, note this may be a no-op"
-            );
-            if let Err(e) = ca_server.renew_objects_all(&actor).await {
-                error!("Background re-issuing of about to expire objects failed: {}", e);
+    /// Queues tasks for background jobs when the server is started
+    async fn queue_start_tasks(&self) -> KrillResult<()> {
+        // If there are only a few CAs in this krill instance, then we
+        // will just want to re-sync them with their parents and repository
+        // on start up.
+        //
+        // If there are many, then will apply some random delays (jitter)
```

Comment:
s/then will/then/ or s/then will/then we will/

Reply:
ack